src/store/cache/user/hooks.ts

```diff
@@ -285,8 +285,8 @@ export function fetchUser(
     if (setStatus) setStatus(Status.Loading)
     const user = await getUserMetadata(wallet, aud)
 
-    const isStaker = await aud.Staking.isStaker(wallet)
-    if (!isStaker) {
+    const totalStaked = await aud.Staking.totalStakedFor(wallet)
```

Comment:
gotcha so basically instead of checking if they are a staker, we go with how much is staked to determine if they qualify? and the reason for this is some wallets were stakers but now they have nothing staked, but the isStaker bool is still true?

Reply:
Previously we determined it by how much they stake - but we have this new case where the user's page should appear as if they are a node operator even after they remove their nodes because they could still have lingering delegators that would need to undelegate stake